packages/docusaurus-utils/package.json

```
@@ -18,31 +18,38 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@docusaurus/types": "2.0.0-beta.9",
     "@mdx-js/runtime": "^1.6.22",
+    "autoprefixer": "^10.4.0",
     "chalk": "^4.1.2",
+    "css-minimizer-webpack-plugin": "^3.2.0",
     "escape-string-regexp": "^4.0.0",
     "fs-extra": "^10.0.0",
     "globby": "^11.0.4",
     "gray-matter": "^4.0.3",
     "lodash": "^4.17.20",
     "micromatch": "^4.0.4",
+    "mini-css-extract-plugin": "^2.4.5",
     "remark-mdx-remove-exports": "^1.6.22",
     "remark-mdx-remove-imports": "^1.6.22",
     "resolve-pathname": "^3.0.0",
-    "tslib": "^2.3.1"
+    "terser-webpack-plugin": "^5.2.5",
+    "tslib": "^2.3.1",
+    "webpack-merge": "^5.8.0"
   },
   "engines": {
     "node": ">=14"
   },
   "devDependencies": {
+    "@docusaurus/types": "workspace:*",
     "@types/dedent": "^0.7.0",
     "@types/micromatch": "^4.0.2",
     "@types/react-dom": "^17.0.1",
     "dedent": "^0.7.0"
   },
   "peerDependencies": {
+    "@babel/core": ">=7.12.0",
```

this range is not correct, we should not allow babel 8+

```suggestion
    "@babel/core": "^7.12.0",
```